### PR TITLE
Navjeevan kaur

### PR DESCRIPTION
**Pull request by navjeevan24**
Hey @dipanshu42 i did some changes in Returnbook class. Can you please tell to my reviewer @akay08923 to review my code.

### CODE REVIEW

**Review by akay08923 — CHANGES_REQUESTED** [review]
Hi @dipanshu42 , there were some errors in @navjeevan24 's updated code. So i suggested the correct code according to the guidlines. she should have a look again. Once the code has been corrected then let me know i will approve it.
Ankush

**Comment by dipanshu42** [comment]
Okay i'll let her know

**Comment by navjeevan24** [comment]
Hey @dipanshu42 I did correction in code can you please review it from my reviewer @akay08923 thanks.

**Comment by dipanshu42** [comment]
Sure thanks

File: library/returnBook/ReturnBookUI.java [hunk 1]
```diff
@@ -48,20 +48,20 @@ public void Run() {
 				
 			case READY:
 				//String BoOk_InPuT_StRiNg = iNpUt("Scan Book (<enter> completes): ");
-				String bookInputString = input("Scan book (<enter> completes): ");
+				String BookInputString = input("Scan book (<enter> completes): ");
 				//if (BoOk_InPuT_StRiNg.length() == 0) 
-					if (Book_Input_String.Length() ==0)
+					if (bookInputString.Length() ==0)
 					//CoNtRoL.sCaNnInG_cOmPlEtE();
-					Control. Scanning_Complete();
```
[the remark this answers is not in the document]
Comment (navjeevan24) [reply library/returnBook/ReturnBookUI.java:59]: i idid it

**Review by akay08923 — APPROVED** [review]

**Comment by dipanshu42** [comment]
Should she merge?

**Comment by navjeevan24** [comment]
ok I will merge it.

**Pull request merged**